source/P0323R9.bs

```diff
@@ -1003,8 +1008,8 @@ void swap(expected<T, E>& rhs) noexcept(see below);
   * Lvalues of type `E` are `Swappable`; and
   * `is_void_v<T>` is `true` or `is_void_v<T>` is `false` and either:
 
-    * `is_move_constructible_v<T>` is `true`; or
-    * `is_move_constructible_v<E>` is `true`.
+    * `is_nothrow_move_constructible_v<T>` is `true`; or
+    * `is_nothrow_move_constructible_v<E>` is `true`.
```

Comment:
the "otherwise, the function is not defined" item requires either `T` or `E` to be nothrow-move-constructible.